roles/kubernetes/control-plane/templates/podsecurity.yaml.j2

```diff
@@ -1,5 +1,5 @@
 {% if kube_pod_security_use_default %}
-apiVersion: pod-security.admission.config.k8s.io/v1beta1
+apiVersion: pod-security.admission.config.k8s.io/v1
```

Comment:
according to https://github.com/kubernetes/website/blob/9a0d240bb54395ed83cfa4cf434f3912a03a424e/content/en/docs/tutorials/security/cluster-level-pss.md?plain=1#L214

``pod-security.admission.config.k8s.io/v1` configuration requires v1.25+.`  and the minimum version Kubespray supports is v1.25.0.
So it is fine to change here as this.